### PR TITLE
adding info about installation on RHEL/FEDORA.

### PR DESCRIPTION
**Pull request by vmindru**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,7 +66,7 @@ files, python code (modules, plugins) and playbooks.
   - per-role repository
   - roles with sub-roles
   - per-playbook repository
-* It should work with rolesfiles and with local roles.
+* It should work with roles files and with local roles.
```
Comment (willthames) [inline README.md]: This should probably be roles requirements files these days

File: README.md [hunk 2]
```diff
@@ -15,6 +15,24 @@ export PYTHONPATH=$PYTHONPATH:`pwd`/ansible-review/lib
 export PATH=$PATH:`pwd`/ansible-review/bin
 ```
 
+## Fedora
+
+ansible-review can be found in the standard base Fedora repo.
+
+```
+yum install ansible-review
```
Comment (willthames) [inline README.md]: dnf? ;)

File: README.md [hunk 3]
```diff
@@ -15,6 +15,24 @@ export PYTHONPATH=$PYTHONPATH:`pwd`/ansible-review/lib
...
+## on RHEL
+
+ansible-review also can be installed from EPEL repo
+
+```
+yum install epel-release
+yum install ansible-review
+```
+
+Make sure to have optional repos [enabled](http://fedoraproject.org/wiki/EPEL#How_can_I_use_these_extra_packages.3F)
```
Comment (willthames) [inline README.md]: You can probably merge these two sections into one - talk about the epel prerequisites first, and then just talk about yum install ansible-review for both

File: README.md (hunk 2 above)
Comment (vmindru) [reply README.md]: old sys admin has hard time to change old habits :-D, they should remove the yum alias :)

File: README.md (hunk 3 above)
Comment (vmindru) [reply README.md]: will do

File: README.md (hunk 1 above)
Comment (vmindru) [reply README.md]: so should this line go away completely? or what should in contain ?

Comment (willthames) [reply README.md]: `* It should work with roles requirement files and with local roles` should be fine.

**Comment by vmindru** [comment]
@willthames  done

**Comment by willthames** [comment]
Tests are failing because Ansible devel no longer supports python 3.4

Can you change py34-ansibledevel to py35-ansibledevel in tox.ini

```
envlist = py27-ansible{19,20,21,22,23,24,devel},py34-ansible{22,devel},flake8
```

to

```
envlist = py27-ansible{19,20,21,22,23,24,devel},py35-ansible{22,23,24,devel},flake8
```

should do the trick

**Comment by vmindru** [comment]
Ack, will do.

**Comment by willthames** [comment]
Thanks @vmindru

**Pull request merged**